api/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "API for the Semantic Scholar Reader",
   "main": "index.js",
   "scripts": {
-    "start": "ts-node-dev src/index.ts",
-    "start:prod": "node build/index.js",
+    "start": "ts-node-dev --max-http-header-size=65536 src/index.ts",
```

Comment:
My only thought is because this is in json we can't comment about why it's being set, consider perhaps moving it into a bash script that can have a note?

Reply:
I dunno, I like the simplicity of keeping it in package.json since it's just one command. No need to open a separate file just to see what `start` is actually doing (and I could see someone later being like "why do we have this one command in a shell script by itself?" and submitting a PR to inline it here).